src/noisepy/seis/plotting_modules.py

```diff
@@ -860,7 +861,7 @@ def plot_all_moveout_1D_1comp(
         plt.title("%s %s filtered @%4.1f-%4.1f Hz" % (sta, ccomp, freqmin, freqmax))
         plt.xlabel("time (s)")
         plt.ylabel("offset (km)")
-        plt.text(disp_lag * 0.9, dist + 0.5, receiver, fontsize=6)
+        plt.text(disp_lag * 0.9, dist + 0.5, treceiver, fontsize=6)
```

Comment:
Now the names of receiver stations instead of source stations will be shown on the plotted figure.